manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/ConnectorWorkerTest.java

```diff
@@ -157,4 +173,21 @@ void handleWorkDeletingWithKnownResourceMultiplePasses(ManagedResourceStatus sta
         verify(connectorEntity, atLeastOnce()).setModifiedAt(any(ZonedDateTime.class));
     }
 
+    private static Stream<Arguments> provideArgsForCreateTest() {
+        return Stream.of(
+                Arguments.of(ManagedResourceStatus.ACCEPTED, ConnectorState.READY, ManagedResourceStatus.READY, ManagedResourceStatus.READY),
+                Arguments.of(ManagedResourceStatus.ACCEPTED, ConnectorState.FAILED, ManagedResourceStatus.FAILED, ManagedResourceStatus.PROVISIONING),
```

Comment:
I think, although not strictly the reported issue, we should add `connectorEntity.setDependencyStatus(ManagedResourceStatus.FAILED)` [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/95d86217d08259695877efdc724872d083f65119/manager/src/main/java/com/redhat/service/bridge/manager/workers/resources/ConnectorWorker.java#L108) (after L#108 before L#109) for consistency. This state is reached when we the MC is `FAILED` and IMO it makes more sense to set our `Connector` Dependency Status to `FAILED` too. At the moment, as these tests show, it remains in `PROVISIONING`. WDYT?

Reply:
It would also then simplify the test as both `Connector.status` and `Connector.dependencyStatus` are the same after the `Work` completes.

Reply:
I totally agree

Reply:
Done :+1: